### PR TITLE
Ran dotnet format instead of just style

### PR DESCRIPTION
**Pull request by RageCage64**
I had only run dotnet format style and not dotnet format itself, which
caught more things. Also fixed a thing on JudgementViewModel to get rid
of a nullable warning.

### CODE REVIEW

File: LeaderboardBackend/Models/ViewModels/JudgementViewModel.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ public record JudgementViewModel
 	///   When the judgement was made. Follows <a href="https://www.ietf.org/rfc/rfc3339.txt">RFC 3339</a>.
 	/// </summary>
 	/// <example>2022-01-01T12:34:56Z / 2022-01-01T12:34:56+01:00</example>
-	public string CreatedAt { get; set; }
+	public string? CreatedAt { get; set; }
```
Comment (zysim) [inline LeaderboardBackend/Models/ViewModels/JudgementViewModel.cs:20]: A little iffy about this one since `CreatedAt` shouldn't ever be `null`

**Review by zysim — COMMENTED** [review]
Just thought of something extra;

On top of reviewing our style guide, we should add `dotnet format` to our CI. *Possibly* with `--exclude Migrations`.

**Comment by zysim** [comment]
> Just thought of something extra;
> 
> On top of reviewing our style guide, we should add `dotnet format` to our CI. _Possibly_ with `--exclude Migrations`.

With the former, frankly we don't have to do it like *now* now, since there *are* lots of rules in em. Could always update the style guide if a future conflict pops up. And I can add the build step tomorrow if you don't already do it.

File: LeaderboardBackend/Models/ViewModels/JudgementViewModel.cs (hunk 1 above)
Comment (RageCage64) [reply LeaderboardBackend/Models/ViewModels/JudgementViewModel.cs:20]: It doesn't really matter, cause this is an output model. We don't do any data validation on it.

**Comment by RageCage64** [comment]
The migrations being formatted does not appear to have any adverse affects, considering the tests all still work as expected.

File: LeaderboardBackend/Models/ViewModels/JudgementViewModel.cs (hunk 1 above)
Comment (zysim) [reply LeaderboardBackend/Models/ViewModels/JudgementViewModel.cs:20]: Mmmmmmmmmm fine

**Comment by zysim** [comment]
Oh yeah I'm sure everything works. In fact I expect it to too. I'm just probably thinking too far ahead where it might slow down builds when we have a lot of migration files.

**Review by zysim — APPROVED** [review]

**Pull request merged**